app/display.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
+
+from app.book import Book
+
+
+class Display(ABC):
+    @staticmethod
+    @abstractmethod
+    def display(book: Book) -> None:
+        pass
+
+
+class DisplayConsole(Display):
```

Comment:
The mentor asked to rewrite init method to be able to pass Book class instance
Something like this:

def __init__(self, book: Book) -> None:
        self._book = book

    def display(self) -> None:
        print(self._book.content)